wgsl/index.bs

```diff
@@ -12515,7 +12674,45 @@ Note: The original value is always within range of the destination type when
 the original type is one of [=i32=] or [=u32=] and the destination type is [=f32=].
 
 Note: The original value is always within range of the destination type when
-the source type is a floating point type with fewer exponent and mantissa bits than the target floating point type.
+the source type is a floating point type with fewer exponent and significand bits than the target floating point type.
+
+### Domains of Floating Point Expressions and Built-in Functions ### {#domains-of-floating-point-expressions-and-builtins}
+
+Previous sections describe the expected behavior when a floating point expression is evaluated
+outside its [=domain=].
+
+Sections [[#arithmetic-expr]] and [[#numeric-builtin-functions]] define the [=domains=] for floating point expressions and built-in functions, respectively.
+If no restriction is listed for a given operation, then the domain is total: the domain includes all finite and infinite inputs.
+Otherwise an explicit domain is listed.
+
+In many cases where a WGSL operation corresponds to an operation defined by [[!IEEE-754|IEEE-754]], they will have the same domain.
+For example the both the WGSL and IEEE-754 `acos` operations have a domain of [&minus;1,1].
+
+For a [=component-wise=] WGSL operation with an explicitly listed domain, only the scalar case is described. The vector case is inferred from the component-wise semantics.
+
+Some WGSL operations may be implemented in terms of other WGSL expressions.
+Section [[#floating-point-accuracy]] lists these as having [=inherited from=] accuracy.
+When listing the domain for one of these operations, either:
+
+* The domain is explicitly described, or
+* The domain is stated as <dfn noexport>implied from linear terms</dfn>, meaning the domain
+    is derived by:
+     * Assuming the original operation was replaced by the "inherited from" expression, which is a combination of floating point addition, subtraction, and mulitiplication operations.
+     * Applying and combining the domain restrictions for those remaining operations over the given parameters.
+
+For example: The `dot(a,b)` function over 2-element vectors |a| and |b| has its accuracy [=inherited from=] the expression
+    |a[0]| * |b[0]| + |a[1]| * |b[1]|.
```

Comment:
This doesn't render like |a| and |b| previous to it. It does work below so I think you want:
```suggestion
    |a|[0] * |b|[0] + |a|[1] * |b|[1].
```